doc-content/enterprise-only/installation/smart-router-config-file-proc.adoc

```diff
@@ -0,0 +1,106 @@
+[id='smart-router-config-file-proc_{context}']
+= Configuring Smart Router settings using a configuration file
+
+Instead of configuring Smart Router settings in the command line, you can use a configuration file. In this case, settings, including any passwords, are not visible in the command line terminal and in server logs.
```

Comment:
```suggestion
Instead of configuring Smart Router settings in the command line, you can use a configuration file. In this case, settings, including any passwords, are not visible in the command line terminal and server logs.
```
Unnecessary word.